app/models/user.rb

```diff
@@ -565,6 +565,53 @@ def reactions_to
     Reaction.for_user(self)
   end
 
+  def latest_activity
+    return unless registered == true
+
+    latest_activity = last_comment_at || last_article_at || latest_article_updated_at ||
+      last_reacted_at || profile_updated_at || last_moderation_notification
+
+    if latest_activity.strftime("%Y-%m-%d") == Time.zone.today.strftime("%Y-%m-%d")
+      "Today #{latest_activity.strftime('%b, %Y')}"
+    elsif latest_activity.strftime("%Y-%m-%d") == Date.yesterday.strftime("%Y-%m-%d")
+      "Yesterday #{latest_activity.strftime('%b, %Y')}"
+    else
+      latest_activity.strftime("%d %b, %Y")
+    end
+  end
+
+  def current_status
+    if suspended?
+      "Suspended"
+    elsif comment_suspended?
+      "Comment suspended"
+    elsif warned?
+      "Warned"
+    elsif trusted?
+      "Trusted"
+    else
+      "Good standing"
+    end
+  end
+
+  def current_roles
+    if admin?
+      "Admin"
+    elsif super_admin?
+      "Super Admin"
+    elsif tech_admin?
+      "Tech Admin"
+    elsif tag_moderator?
+      "Moderator"
+    else
+      "Resource Admin"
```

Comment:
I think this means people with no special roles are being displayed as 'resource admins'?

Reply:
Great catch; fixed! 🤦‍♀️